pyramid/scripts/pcreate.py

```diff
@@ -99,9 +106,13 @@ def output_path(self):
     def project_vars(self):
         output_dir = self.output_path
         project_name = os.path.basename(os.path.split(output_dir)[1])
-        pkg_name = _bad_chars_re.sub(
-            '', project_name.lower().replace('-', '_'))
-        safe_name = pkg_resources.safe_name(project_name)
+        if self.options.package_name is None:
+            pkg_name = _bad_chars_re.sub(
+                '', project_name.lower().replace('-', '_'))
```

Comment:
Where is the assumption of safety documented? I don't get that from reading the help. Maybe rephrase that sentence slightly because I think `has to be a valid python package name` implies that it will be validated. Perhaps `Assumed to be a valid python package name`.